lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -597,23 +617,29 @@ def test_student_progress(self):
 
 
 @ddt.ddt
-class CCXCoachTabTestCase(ModuleStoreTestCase):
+class CCXCoachTabTestCase(SharedModuleStoreTestCase):
     """
     Test case for CCX coach tab.
     """
+    @classmethod
+    def setUpClass(cls):
+        super(CCXCoachTabTestCase, cls).setUpClass()
+        cls.ccx_enabled_course = CourseFactory.create(enable_ccx=True)
```

Comment:
I assume there's not some 3rd state called "ccx not enabled" (different from ccx disabled) ?